src/analysis/parsimonious_flux_balance_analysis.jl

```diff
@@ -43,8 +43,7 @@ if something went wrong).
 # Example
 ```
 model = load_model("e_coli_core.json")
-optmodel = parsimonious_flux_balance_analysis(model, biomass, Gurobi.Optimizer)
-value.(solution[:x])  # extract the flux from the optimizer
+parsimonious_flux_balance_analysis(model, biomass, Gurobi.Optimizer) |> values_vec
```

Comment:
Need to remove Gurobi from here and replace with Clarabel (unless your CI has Gurobi installed?)